doc/source/user_guide/text.rst

```diff
@@ -641,21 +641,35 @@ You can check whether elements contain a pattern:
 .. ipython:: python
 
    pattern = r'[0-9][a-z]'
-   pd.Series(['1', '2', '3a', '3b', '03c'],
+   pd.Series(['1', '2', '3a', '3b', '03c', '4dx'],
              dtype="string").str.contains(pattern)
 
 Or whether elements match a pattern:
 
 .. ipython:: python
 
-   pd.Series(['1', '2', '3a', '3b', '03c'],
+   pd.Series(['1', '2', '3a', '3b', '03c', '4dx'],
              dtype="string").str.match(pattern)
 
-The distinction between ``match`` and ``contains`` is strictness: ``match``
-relies on strict ``re.match``, while ``contains`` relies on ``re.search``.
+.. versionadded:: 1.1.0
 
-Methods like ``match``, ``contains``, ``startswith``, and ``endswith`` take
-an extra ``na`` argument so missing values can be considered True or False:
+.. ipython:: python
+
+   pd.Series(['1', '2', '3a', '3b', '03c', '4dx'],
+             dtype="string").str.fullmatch(pattern)
+
+
+The distinction between ``match``, ``fullmatch``, and ``contains`` is strictness:
```

Comment:
can you link back to python's re package here, I think you can do ```:ref:`re.fullmatch` ``` should work (build this doc-page and see)

Reply:
Thanks for the feedback. Putting hyperlinks there is a great idea. It took me a few tries to get it working, but my commit 5ff5203 from last night looks to be generating these links successfully. The syntax that I found that worked was:
```
`text of hyperlink <https://url.goes.here>`_
```
The need for a trailing underscore really threw me for a loop for a while. Anyhow, this issue should be fixed now. Let me know if there's a different way I should put those links in.